apps/studio/src/schemas/page.ts

```diff
@@ -43,8 +33,9 @@ export const createPageSchema = z.object({
     .max(MAX_PAGE_URL_LENGTH, {
       message: `Page URL should be shorter than ${MAX_PAGE_URL_LENGTH} characters.`,
     }),
-  // TODO: add the actual layouts in here
-  layout: z.enum(PAGE_LAYOUTS).default("content"),
+  layout: z
+    .enum<string, [LayoutValues]>(LAYOUT_VALUES as [LayoutValues])
```

Comment:
have we tried using `satisfies` rather than `as`?

Reply:
cannot, we need to cast. `LayoutValues[]` does not satisfy `[LayoutValues]` haha